sdk/storage/azblob/appendblob/client_test.go

```diff
@@ -125,6 +125,36 @@ func (s *AppendBlobRecordedTestsSuite) TestAppendBlock() {
 	_require.Equal(*appendResp.BlobCommittedBlockCount, int32(2))
 }
 
+func (s *AppendBlobRecordedTestsSuite) TestAppendBlockHighThroughput() {
+	_require := require.New(s.T())
+	testName := s.T().Name()
+	svcClient, err := testcommon.GetServiceClient(s.T(), testcommon.TestAccountDefault, nil)
+	_require.NoError(err)
+
+	containerName := testcommon.GenerateContainerName(testName)
+	containerClient := testcommon.CreateNewContainer(context.Background(), _require, containerName, svcClient)
+	defer testcommon.DeleteContainer(context.Background(), _require, containerClient)
+
+	abName := testcommon.GenerateBlobName(testName)
+	abClient := containerClient.NewAppendBlobClient(testcommon.GenerateBlobName(abName))
+
+	// Create AppendBlob with 5MB data
+	_, err = abClient.Create(context.Background(), nil)
+	_require.Nil(err)
+	contentSize := 5 * 1024 * 1024 // 5MB
```

Comment:
Wouldn't the recording be very large if we store 5 MB of content?
We can make the content size smaller or make this test unrecorded.

Reply:
The appendblob size has to be >4 MB to test this feature out. I can make it unrecorded though!